Revolution Rotation Calculator.py

```diff
@@ -514,18 +498,18 @@ def AdjustCooldowns(Current_Buff, Adrenaline, Time): # Decreases Cooldowns of ab
         if Runthrough == 10000:
             End_Estimation = int(Time_Remaining_Calculation * (time.time() - Start))
         if Runthrough % 10000 == 0:
-            print(str('\r===== ') + str(round(float(Runthrough/Permutations) * 100, 3)) + str('% ===== ') + str('Estimated time remaining: ') + str(Get_Time(int(End_Estimation - (time.time() - Start)))) + '; Best found: ' + str(CurrentHighest) + '%' +
+            # print(f'\r===== {round(float(Runthrough/Permutations) * 100, 3)}% ===== Estimated time remaining: {Get_Time(int(End_Estimation - (time.time() - Start)))}; Best found: {CurrentHighest}%' + (' ' * 22), end = '')
+            print('\r===== ' + str(round(float(Runthrough/Permutations) * 100, 3)) + '% ===== Estimated time remaining: ' + str(Get_Time(int(End_Estimation - (time.time() - Start)))) + '; Best found: ' + str(CurrentHighest) + '%' +

```

Comment:
What does `(' ' * 22)` do?

Reply:
the ' '*22 (even though I think 22 can be reduced) just means that when \r is used you won't get the previous line one percentage written slightly further past the current (if that makes sense). For example: if I had 'abcd' on console, and wanted to print \r abc (so printing abc on the same line) I would still get *abc*d so a simple fix to this would be print('\r abc ') (space at the end).